packages/shared-components/src/Histogram.tsx

```diff
@@ -242,27 +264,32 @@ function Histogram({
                   stroke={CHART_SOLUTION_COLORS.correct}
                   fill={CHART_SOLUTION_COLORS.correct}
                   enableBackground="#FFFFFF"
-                  label={{
-                    fill: CHART_SOLUTION_COLORS.correct,
-                    position: 'top',
-                    value: 'Korrekt',
-                  }}
+                  opacity={basic ? 0.6 : 1}
+                  label={
+                    !basic && {
+                      fill: CHART_SOLUTION_COLORS.correct,
+                      position: 'top',
+                      value: 'Korrekt',
+                    }
+                  }
                   className={textSize}
                 />
               )
             )}
         </BarChart>
       </ResponsiveContainer>
 
-      <div className="flex flex-row items-center float-right gap-2 mr-4">
-        <div className="font-bold">Bins:</div>
-        <input
-          className="rounded-md"
-          type="number"
-          value={numBins}
-          onChange={(e) => setNumBins(+e.target.value)}
-        />
-      </div>
+      {!hideBins && (
```

Comment:
This is very specific to the evaluation. Long-term, we should probably have the Histogram only get numBins from props, and extract and manage this on the evaluation components.